Before: ecb07501b873ab031a49801b6b9f9237a53a2b3d9a70dec2d1bb3521975247ea
After: b165eb1ae8e07cdd8105c8120e47aa1384f92032a5035cd228cc84f501a6644d
Patch: b547a2e3ce9c627a905ef96857833c857a6e12dd3b5a73c07cecd843aba873aa
--- corvid-features/corvid.rb	2012-07-25 11:10:18.354651596 +1000
+++ /dev/null	2012-07-25 08:15:55.423554813 +1000
@@ -1,17 +0,0 @@
-def install
-  empty_directory '.corvid'
-  copy_file       '.corvid/Gemfile'
-  copy_file       '.gitignore'
-  copy_file       '.simplecov'
-  copy_file       '.yardopts'
-  copy_file       'CHANGELOG.md'
-  copy_file       'Gemfile'
-  copy_file       'Guardfile'
-  copy_file       'Rakefile'
-  copy_file       'README.md'
-  copy_executable 'bin/rake'
-  copy_executable 'bin/yard'
-  copy_executable 'bin/yardoc'
-  copy_executable 'bin/yri'
-  empty_directory 'lib'
-end
--- corvid-features/test_spec.rb	2012-07-25 11:10:18.777982162 +1000
+++ /dev/null	2012-07-25 08:15:55.423554813 +1000
@@ -1,8 +0,0 @@
-def install
-  copy_file               '.rspec'
-  copy_executable         'bin/guard'
-  copy_executable         'bin/rspec'
-  copy_file_unless_exists 'test/bootstrap/all.rb'
-  copy_file               'test/bootstrap/spec.rb'
-  empty_directory         'test/spec'
-end
--- corvid-features/test_unit.rb	2012-07-25 11:10:18.824648524 +1000
+++ /dev/null	2012-07-25 08:15:55.423554813 +1000
@@ -1,6 +0,0 @@
-def install
-  copy_executable         'bin/guard'
-  copy_file_unless_exists 'test/bootstrap/all.rb'
-  copy_file               'test/bootstrap/unit.rb'
-  empty_directory         'test/unit'
-end
